library/ssl_tls.c

```diff
@@ -4685,6 +4689,93 @@ const char *mbedtls_ssl_get_alpn_protocol( const mbedtls_ssl_context *ssl )
 }
 #endif /* MBEDTLS_SSL_ALPN */
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+#if defined(MBEDTLS_DEBUG_C)
+const char *mbedtls_ssl_get_srtp_profile_as_string ( mbedtls_ssl_srtp_profile profile )
```

Comment:
Blocker: This requires a declaration in `ssl.h`. Alternatively, you could make it `static inline` function, declared and defined in `ssl.h`.